features/filmStrip/components/native/FilmStripContainer.js

```diff
@@ -14,8 +14,21 @@ export class FilmStripContainer extends Component {
      * @returns {ReactElement}
      */
     render() {
+        // The following property is responsible to hide/show the toolbar
+        // view by moving it out of site of the screen boundaries.
+        // An attempt to use the opacity property has been made in order
+        // to eventually implement a fadeIn/fadeOut animation, however a known
+        // react native problem has been doscovered, which allows the view to
+        // still capture touch events even if hidden. Alternatives will be
+        // investigated.
+        var bottomValue = styles.filmStrip.bottom;
+        if (!this.props.isVisible)
+            bottomValue = -Dimensions.get('window').height;
```

Comment:
That's practically an if/else or a ?: which I find to be with cleaner/easier to grasp meaning. Again, this is something I'll take care of.